docs/40-troubleshooting.md

```diff
@@ -1,4 +1,47 @@
-# Getting logs without Azure Portal
+## Debugging and Troubleshooting
+
+### After creating my cluster, my nodes go to a 'startTaskFailed' state. Why?
+The most common case for this is that there was an issue with package installation or the custom script failed to run. To troubleshoot this you can simply download the output logs from the node.
+
+```r
+cluster <- makeCluster('myConfig.json')
+...
+# Get standard error file
+getClusterFile(cluster, "tvm-1170471534_2-20170829t072146z", "stderr.txt", downloadPath = "pool-errors.txt")
```

Comment:
just to be super obvious, we should also remind people how to get their node id (in this case: tvm-1170471534_2-20170829t072146z)